Xamarin.Forms.ControlGallery.iOS/AppDelegate.cs

```diff
@@ -159,7 +159,7 @@ public override bool FinishedLaunching(UIApplication uiApplication, NSDictionary
 			App.IOSVersion = int.Parse(versionPart[0]);
 
 			Xamarin.Calabash.Start();
-			
+			// Forms.SetFlags("CollectionView_Experimental", "Visual_Experimental");
```

Comment:
Should we `#if` this instead of commenting it out?

Reply:
I mainly added this so it's just easier for anyone using ControlGallery to uncomment these and test the experimental.

Reply:
👍